Allwmake

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-cd ${0%/*} || exit 1    # Run from this directory
+set -e -u
```

Comment:
`set -e` won't work since it directly exits. For me it results in 
`./Allwmake: 51: CPLUS_INCLUDE_PATH: parameter not set`

Reply:
I added some placeholders so that this error is not triggered at expected places. By the way, this is because of `-u`, not because of `-e`.